frontend/src/views/manager/ConceptList.js

```diff
@@ -282,6 +295,12 @@ const ConceptList = ({
           onChange={evt => setConceptFilter(evt.target.value)}
           className={classes.filterText}
         />
+        {conceptTags.length > 0 && <IconButton onClick={openMenu}><ExpandMore /></IconButton>}
+        <Menu anchorEl={menu.anchor} open={menu.open} onClose={closeMenu}>
+          {conceptTags.map(tag =>
+            <MenuItem key={tag.id} onClick={selectMenu(tag.label)}>tag:{tag.label}</MenuItem>
```

Comment:
Tag list items shouldn't have the word 'tag' in it. Tooltip on the filter menu would do the trick (e.g. Filter by tags)
```suggestion
            <MenuItem key={tag.id} onClick={selectMenu(tag.label)}>{tag.label}</MenuItem>
```